lib/telephone_number/class_methods.rb

```diff
@@ -1,5 +1,21 @@
 module TelephoneNumber
   module ClassMethods
+    attr_accessor :override_file, :default_format_string
+    attr_reader :default_format_pattern
+
+    # allows users to override the default data
+    @override_file = nil
+
+    # allows users to provide a default format pattern
+    @default_format_pattern = nil
+
+    # allows users to provide a default format string
+    @default_format_string = nil
+
+    def default_format_pattern=(format_string)
```

Comment:
Wouldn't `attr_writer` assume it would be given 

```ruby
def self.default_format_pattern=(format_string)
  @default_format_pattern = format_string
end
```